tests/ja4-quic/test.yaml

```diff
@@ -0,0 +1,14 @@
+requires:
+  min-version: 7.0.0
+  files:
+    - rust/src/quic/ja4.rs
```

Comment:
ok. They're likely all skipped for now.

Reply:
Yep, that file does not exist until that PR.